cockatrice/src/abstractcarddragitem.h

```diff
@@ -7,25 +7,43 @@ class QGraphicsScene;
 class CardZone;
 class CardInfo;
 
-class AbstractCardDragItem : public QObject, public QGraphicsItem {
+class AbstractCardDragItem : public QObject, public QGraphicsItem
+{
     Q_OBJECT
     Q_INTERFACES(QGraphicsItem)
-protected:
+    protected:
     AbstractCardItem *item;
     QPointF hotSpot;
     QList<AbstractCardDragItem *> childDrags;
-public:
-    enum { Type = typeCardDrag };
-    int type() const { return Type; }
+
+    public:
+    enum
+    {
+        Type = typeCardDrag
+    };
+    int type() const
+    {
+        return Type;
+    }
     AbstractCardDragItem(AbstractCardItem *_item, const QPointF &_hotSpot, AbstractCardDragItem *parentDrag = 0);
     ~AbstractCardDragItem();
-    QRectF boundingRect() const { return QRectF(0, 0, CARD_WIDTH, CARD_HEIGHT); }
+    QRectF boundingRect() const
+    {
+        return QRectF(0, 0, CARD_WIDTH, CARD_HEIGHT);
+    }
     void paint(QPainter *painter, const QStyleOptionGraphicsItem *option, QWidget *widget);
-    AbstractCardItem *getItem() const { return item; }
-    QPointF getHotSpot() const { return hotSpot; }
+    AbstractCardItem *getItem() const
+    {
+        return item;
+    }
+    QPointF getHotSpot() const
+    {
+        return hotSpot;
+    }
     void addChildDrag(AbstractCardDragItem *child);
     virtual void updatePosition(const QPointF &cursorScenePos) = 0;
-protected:
+
+    protected:
```

Comment:
Agree with tabbing in the word, but can we then tab in all the components so they’re one further indent from the protection status

Reply:
That's farther away from our current code style, and also I think less common in general. I don't have strong feelings aside from those remarks

Reply:
Removed.